### ISSUE TITLE
[ BUG ] 400 error with 'Edit-FalconDetection' when 'comment' is present without 'status'

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#135]
**Describe the bug**
`400: Failed to validate resource` is produced when an `Edit-FalconDetection` request is sent with a `comment`  and no `status` value.

**To Reproduce**
```
Edit-FalconDetection -Ids <id>, <id> -Comment 'hello world'
```
**Expected behavior**
The API does not allow `comment` without `status`. Produce an error message so the user understands what they need to do to correct the problem.

**Environment (please complete the following information):**
 - OS: Windows 10
 - PowerShell: 5.1, 7.1.4
 - PSFalcon: 2.1.x

**Comment by bk-cs** [comment]
This has been fixed for the next release. You can resolve this issue with your local module before the next release by replacing `Public\detects.ps1` with the contents of this file: https://raw.githubusercontent.com/CrowdStrike/psfalcon/master/Public/detects.ps1

**Comment by bk-cs** [comment]
Resolved with v2.1.6 release.
